### DISCUSSION TITLE
Adding more metrics

### DISCUSSION THREAD

**Discussion by vebrt99** [GoogleChrome/lighthouse#15672] [Q&A] [answered]
Hello,

I would like to ask how can I add more metrics into Lighthouse. I have found folder with these metrics (CLS, FCP, FMP, INP, TTI, LCP, Max Potential FID, SI, TBT), but i dont know how to use them all and also adjust the weight of metrics for calculating Performance Score (with adding other metrics to this calculation). Can anyone help me with this (if it is even possible)?

I do have need it for my thesis.

**Comment by adamraine** [comment, marked as answer]
You can control which audits are visible in the report and how they contribute to the performance score in the config. Please see https://github.com/GoogleChrome/lighthouse/blob/main/docs/configuration.md#categories-objectundefined
